### ISSUE TITLE
Crossterm is at version 0.25.0 now

### ISSUE THREAD

**Issue by generic-user1** [Canop/coolor#1]
Crossterm integration fails when using the latest version of crossterm (0.25.0 at time of writing).
I'm hoping this is a simple Cargo.toml change... if not, maybe I can open a PR.

**Comment by Canop** [comment]
What kind of failure do you have ? Can you give me a reproduction case to help clarify the problem ?

**Comment by generic-user1** [comment]
The failure is triggered by attempting to cast a `coolor::Color` into a `crossterm::style::Color` with the `Into` trait.

When using crossterm version 0.23.1, the project will compile and function as intended.

When using crossterm version 0.25.0, the project will not compile. The error message reads:
```
error[E0277]: the trait bound `crossterm::style::Color: From<coolor::Color>` is not satisfied
 --> nonworking\src\main.rs:7:52
  |
7 |     let crossterm_color: crossterm::style::Color = coolor_color.into();
  |                                                    ^^^^^^^^^^^^ ---- required by a bound introduced by this call
  |                                                    |
  |                                                    the trait `From<coolor::Color>` is not implemented for `crossterm::style::Color`
  |
  = help: the trait `From<(u8, u8, u8)>` is implemented for `crossterm::style::Color`
  = note: required for `coolor::Color` to implement `Into<crossterm::style::Color>`
```
I've created a repo that demonstrates this if you'd like to see for yourself:
https://github.com/generic-user1/coolor-error-demo/tree/master

**Comment by Canop** [comment]
Thanks.
I published coolor 0.5.1.
Can you confirm it's OK for you ?

**Comment by generic-user1** [comment]
Yes, this solves this issue on my end. Thanks!
